### ISSUE TITLE
Provide flatpak by default

### ISSUE THREAD

**Issue by cassidyjames** [elementary/os#234]
It looks like our Flatpak-using apps depend on LibFlatpak, but nothing actually pulls in Flatpak itself. The result is that the XDG data dirs are not set up properly, and users can't manage their Flatpaks from Terminal. We should probably just include Flatpak in the base install now.

**Comment by cassidyjames** [comment]
Alternately, AppCenter and Sideload could just depend on Flatpak itself as well as the library.

**Comment by danrabbit** [comment]
Yeah that's probably the cleaner route. Sideload now has runtime depends on flatpak. I'll move this issue to AppCenter
